clusterloader2/testing/experimental/storage/pod-startup/ephemeral-volumes/1_node/configmap/override.yaml

```diff
@@ -0,0 +1,3 @@
+PROVISION_VOLUME: true
+POD_TEMPLATE_PATH: "configmap/pod_with_ConfigMaps.yaml"
+VOLUME_TEMPLATE_PATH: "configmap/ConfigMap.yaml"
```

Comment:
Do you somewhere use the assumption that the file names are capitalized ConfigMap (I don't see that, but maybe I'm missing something).
If not, it would be more natural to have:
configmap/config_map.yaml
configmap/pod_with_config_map.yaml

[Same for secrets and others]

Reply:
There was but no longer, I'll rename it, but volume names are not used as separate words generally, usages are like `ConfigMap`, `EmptyDir` etc. I'll rename them as like `configmap/configmap.yaml`, `configmap/pod_with_configmap.yaml` etc.

Reply:
Sure that sounds fine, thanks!